src/components/Home.js

```diff
@@ -32,13 +36,24 @@ const Home = ()=>{
 
     getUser()
   },[])
-//I got the token back and got the login route to work. Im not sure how I would use verify here. But what im reading is to use jwt decode to get the userid out of the token since I dont see it in the token object
+
   const userLogin = async()=>{
-    const {email, password} = homeData
-    const token = await axios.post('/api/user/login', {email, password})
-    // const verified = await jwt.verify(token, process.env.JWT_SECRET)
-    
-    console.log(token)
+    try {
+      const token = await axios.post('/api/user/login', {email: homeData.email, password: homeData.password})
+      const decodedToken = jwtDecode(token.data)
+      const user = await axios.get(`/api/user/${decodedToken.id}`)
```

Comment:
There should likely be a try / catch here, as we don't know that this will always succeed.

Reply:
A second try/catch?

Reply:
lol, disregard. Not sure what I was thinking. I didn't see the first one. Was probably too late at night! :)